pages/dkp/konvoy/2.2/choose-infrastructure/vsphere/create-capi-vm-image/index.md

```diff
@@ -44,7 +44,11 @@ Using the base OS image created in a previous procedure, DKP creates the new vSp
       distribution_version: "example_7.9"
     ```
 
-1. Create the vSphere VM template with the following command:
+    The website for [Packer](https://www.packer.io/plugins/builders/vsphere/vsphere-clone) gives details around many of these variables. However, there are fewer configuration variables needed for use in the vSphere KIB provisioning than what is listed on the Packer website. Below are the variables and optional configurations to be used with vSphere:
+
+<!Shalin - please enter the variables you believe would benefit the customer>
```

Comment:
-  cluster: vSphere cluster is a name of the collection of ESXi hosts in vSphere
-  datacenter:  vSphere datacenter name. Required if there is more than one datacenter in the vSphere inventory.
-  datastore:  vSphere datastore name. The datastore stores files of the virtual machines. They could be located on a local server hard drive or across the network on a SAN
-  folder:  Virtual machine folder to create the virtual machine in.
-  network: The vSphere network in which the virtual machine will be connected to.
-  resource_pool: vSphere resource pool name. If not set, it will look for the root resource pool of the host or cluster. If a root resource is not found, it will then look for a default resource pool.
-  template:  Name of the vSphere template of the base operating system. konvoy image builder will use the base operating system template and configure it produce new vsphere template. The new vsphere template can be used to create virtual machines for the kubernetes cluster